language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/codeaction/CodeActionUtil.java

```diff
@@ -895,4 +897,31 @@ Optional<T>> getDiagPropertyFilterFunction(int propertyIndex) {
         };
         return filterFunction;
     }
+    
+    public static Optional<Pair<Symbol, Path>> getSymbolAndPath(Diagnostic diagnostic, String diagnosticCode,
```

Comment:
Shall we change the name to something like `findSymbolAndPathForDiagnosticRange()`